03-03-2020_delete_records_award_0000000000.csv

```diff
@@ -0,0 +1,2 @@
+sausage
+rootbeer
```

Comment:
I'm guessing someone was a little hungry and/or thirsty when they created this file...

Reply:
Needed non-numeric values. Who doesn't like food?